rotatescreen/display.py

```diff
@@ -44,41 +28,57 @@ def rotate_to(self, degrees):
         dm.DisplayOrientation = rotation_val
         win32api.ChangeDisplaySettingsEx(self.device, dm)
 
-    def set_landscape(self):
+    def set_landscape(self) -> None:
         self.rotate_to(0)
 
-    def set_landscape_flipped(self):
+    def set_landscape_flipped(self) -> None:
         self.rotate_to(180)
 
-    def set_portrait(self):
+    def set_portrait(self) -> None:
         self.rotate_to(90)
 
-    def set_portrait_flipped(self):
+    def set_portrait_flipped(self) -> None:
         self.rotate_to(270)
 
     @property
-    def current_orientation(self):
-        state = self.devicemodeW.DisplayOrientation
+    def current_orientation(self) -> int:
+        state: int = self.devicemodeW.DisplayOrientation
         return state * 90
 
     @property
-    def info(self):
+    def info(self) -> Dict:
         return win32api.GetMonitorInfo(self.hMonitor)
 
     @property
-    def device(self):
+    def device(self) -> str:
         return self.info["Device"]
 
     @property
-    def is_primary(self):
+    def is_primary(self) -> str:
         # The only flag is MONITORINFOF_PRIMARY which is 1 only for the primary monitor.
         return self.info["Flags"]
 
     @property
-    def device_description(self):
+    def device_description(self) -> Tuple[str, str]:
         display_device = win32api.EnumDisplayDevices(self.device)
         return display_device.DeviceString, display_device.DeviceID
 
     @property
     def devicemodeW(self):
         return win32api.EnumDisplaySettings(self.device, win32con.ENUM_CURRENT_SETTINGS)
+
+
+def get_displays() -> List[Display]:
+    displays: List[Display] = [Display(hMonitor) for hMonitor, _, _ in win32api.EnumDisplayMonitors()]
+    return displays
```

Comment:
Personally I think one of these hints is covered by the return type.

```suggestion
def get_displays() -> List[Display]:
    displays = [Display(hMonitor) for hMonitor, _, _ in win32api.EnumDisplayMonitors()]
    return displays
```